docs/prysm-usage/grafana-dashboard.md

```diff
@@ -0,0 +1,145 @@
+---
+id: grafana-dashboard
+title: Monitoring and alerting with Grafana
+sidebar_label: Monitoring with Grafana
+---
+
+Now that you have your validator and node process running, you will probably need a nice dashboard and alert system to ensure at maximum the profitability of your staking ETH. Here is a simple guide that explain how to get one, without any developer skill.
+
+Here is the result of what you will get if you follow this guide!
+![Grafana dashboard for prysm node and validator](https://github.com/prysmaticlabs/documentation/blob/master/docs/prysm-usage/grafana-dashboards/assets/dashboard_overview.png?raw=true "Grafana dashboard for prysm node and validator")
+
+
+## Metrics from validators and node process
+First thing you need to do, is to add this flag to the end of the validator command
+```
+--enable-account-metrics
+```
+
+Now you have to make sure that you have access to both metrics:
+- on the machine running the node, you will find the node metrics on http://localhost:8080/metrics
+- on the machine running the validator, you will find the validator metrics on http://localhost:8081/metrics
+
+## Prometheus
+### Installation
+[Download Prometheus](https://prometheus.io/download/), then go to the directory where the **prometheus.exe** file is, and edit the **prometheus.yml** file to replace the content of the file by this:
```

Comment:
This can be accomplished by right-click > opening the `.exe`, right? (Windows noob here!)

Reply:
I'm not sure. Also this shouldn't be designed for Windows only, but it's true that .exe is only for Windows. I should find a better to explain this part than using the .exe as a reference